README.md

```diff
@@ -108,6 +108,31 @@ HTML styled "just right".
 Jekyll will print a local URL where the site can be viewed (usually,
 [http://0.0.0.0:4000/](http://0.0.0.0:4000/)).
 
+### Testing using Docker environment 
+
+A containerized development environment can be built using the local Dockerfile. 
+
+```bash
+docker build -t webdev .
+```
+
+The webdev container's run command will run jekyll's serve command with the 
+polling option enabled.  This allows for immediate review of rendered changes.   
```

Comment:
Markdown renders double spaces after a line as a line break. 
I've gone through and swapped out the double spaces for the `\` character instead. 
This should still render a line break in markdown while being a bit nicer to view.

Reply:
Markdown's behavior with the trailing spaces is confusing a bit, I think, because it's not clear if a newline is intended or not. I'm glad there's an alternative syntax for it, but I also don't think we need line breaks in these paragraphs. If the break is important, it's probably worth creating a new paragraph.